dsc/cli.py

```diff
@@ -85,6 +88,62 @@ def reconcile(ctx: click.Context) -> None:
         ctx.exit(1)
 
 
+@main.command()
+@click.pass_context
+@click.option("--sync-data/--no-sync-data", default=False)
+@click.option(
+    "-s",
+    "--source",
+    help=(
+        "Source directory formatted as a local filesystem path or "
+        "an S3 URI in s3://bucket/prefix form"
+    ),
+)
+@click.option(
+    "-d",
+    "--destination",
+    help=(
+        "Destination directory formatted as a local filesystem path or "
+        "an S3 URI in s3://bucket/prefix form"
+    ),
+)
+@click.option(
+    "--dry-run",
+    is_flag=True,
+    help=(
+        "Display the operations that would be performed using the "
+        "sync command without actually running them"
+    ),
+)
```

Comment:
Perhaps these should be prepended with `sync_` for greater clarity?

Reply:
Agreed with @ehanson8 here (or `sync-` if we're going with dashes in CLI args).

Reply:
Yes, `-`!